cellbrowser_tools/validateProcessedImages.py

```diff
@@ -414,8 +421,12 @@ def build_cfe_dataset_2020(prefs):
     # need CellLineName here
     file_infos.rename(columns={"CellLine": "CellLineName"}, inplace=True)
 
+    log.info("Collecting feature data")
+    df_feats = get_csv_features(
```

Comment:
A hard-coded path? If it has to be sure but I'd prefer to see it defined at the top of the file as a global constant.

Reply:
YAY.  I am not anxious to overly engineer this because the whole nature of how data comes in seems to be updated every time around.  The anticipated "next" update will be to have an array of file paths instead of one single one.  No objection to moving it into a single symbolic constant, but it is still the case that it's only used in this one single place in the code.

Reply:
Potentially having a block at the top surrounded by comments saying `# inputs go here` and closed with `# end inputs` then on future work at least it's in one spot.

Reply:
Here I am 2 months later and I'm gonna make that hardcoded file path a symbolic constant now, before I merge this.